### PR TITLE
whkd: Add version 0.1.0

### PR DESCRIPTION
**Pull request by LGUG2Z**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/whkd.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "version": "0.1.0",
+  "description": "A simple hotkey daemon for Windows",
+  "homepage": "https://github.com/LGUG2Z/whkd",
+  "license": "MIT",
+  "suggest": {
+    "komorebi": "extras/komorebi"
+  },
```
Comment (sitiom) [inline bucket/whkd.json:8]:
```suggestion
```
`whkd` can be used as a standalone tool.

File: bucket/whkd.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    }
+  },
+  "bin": "whkd.exe",
+  "checkver": "github",
+  "autoupdate": {
+    "architecture": {
+      "64bit": {
+        "url": "https://github.com/LGUG2Z/whkd/releases/download/v$version/whkd-$version-x86_64-pc-windows-msvc.zip"
+      }
+    }
```
Comment (sitiom) [inline bucket/whkd.json:22]: Add hash extraction
```suggestion
    },
    "hash": {
      "url": "$baseurl/checksums.txt"
    }
```

**Review by sitiom — CHANGES_REQUESTED** [review]
Reformat manifest to 4 spaces.

**Comment by LGUG2Z** [comment]
Reformatted to 4 spaces!

**Review by sitiom — APPROVED** [review]

**Pull request merged**